docs/DeveloperGuide.md

```diff
@@ -6,32 +6,45 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+1. The following sequence diagram shows how the Bye Command works.
+![](ByeCommand.png)
 
 ## Product scope
 ### Target user profile
 
-{Describe the target user profile}
+For airlines counter staff who are fast typist to easily manage flight bookings.
 
 ### Value proposition
 
-{Describe the value proposition: what problem does it solve?}
+Given that the pandemic is ending, more and more people are resuming air travel. <br>
+Therefore, airlines counter staff need a fast an efficient way to manage the many flight bookings.
 
 ## User Stories
 
 |Version| As a ... | I want to ... | So that I can ...|
 |--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
-
+|v1.0|Counter Staff|Add in flight menu item|Add menu items available for in-flight meals|
+|v1.0|Counter Staff|Remove in flight menu item|Remove menu items that can be reserved for in-flight meals|
+|v1.0|Counter Staff|Add in flight meal reservation|Manage in-flight meal reservation request from customers|
+|v1.0|Counter Staff|Remove in flight meal reservation|Manage in-flight meal reservation request from customers|
+|v1.0|Counter Staff|List all in flight menu items|To answer customer queries on the menu items available|
+|v1.0|Counter Staff|Add Flight route|Add new flight route that the airline releases|
+|v1.0|Counter Staff|Remove Flight route|Remove the flight routes the airline will no longer offer|
+|v1.0|Counter Staff|List all flight route|Have an overview of available flight routes|
+|v1.0|Counter Staff|Find flight route|To answer customer's query on whether a flight route exists|
 ## Non-Functional Requirements
 
-{Give non-functional requirements}
+1. The program should not crash under any circumstances.
```

Comment:
It would be helpful if you could explain a bit more on why these requirements are important.